package.json

```diff
@@ -3,7 +3,8 @@
   "version": "2.2.1",
   "description": "eslint plugins to support our JS Code Standards. See @silvermine/eslint-config-silvermine",
   "scripts": {
-    "test": "./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha -- -R spec 'tests/**/*.test.js'"
+    "test": "check-node-version --npm 6.13.4 && ./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha -- -R spec 'tests/**/*.test.js'",
```

Comment:
@pbredenberg I just realized that this won't work. We need to have `./node_modules/.bin/check-node-version`. Could you submit a PR to fix that on any of our silvermine repos that have this issue?

Reply:
Why do we need `./node_modules/.bin`? Since this is an NPM script `./node_modules/.bin` gets added to the PATH automatically.

https://docs.npmjs.com/cli/run-script#description
> In addition to the shell’s pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix. For example, if there is a devDependency on tap in your package, you should write:

Reply:
Ah, @jimjenkins5 is right. I had an error that made it look like this was the culprit, but it was something else actually. Sorry @pbredenberg